Weaver/RealmWeaver.Fody/ModuleWeaver.Schema.cs

```diff
@@ -151,4 +159,9 @@ private MethodDefinition FindOrCreateCctor(TypeDefinition moduleClass)
 
         return cctor;
     }
+
+    private static bool ShouldInclude(ICustomAttributeProvider provider)
+    {
+        return provider.CustomAttributes.All(a => a.AttributeType.Name != "ExplicitAttribute");
```

Comment:
I think we should add an imported reference for `ExplicitAttribute` and compare against that. Otherwise we risk mistaking this for any other type with the same name but in a different namespace.

Comparing against the type name was part of the original implementation, but we ought to be safer now that this feature is public.

Reply:
Good catch, and especially with Explicit being somewhat generic. Will update it.